Todo.WillowBear/Todo.WillowBear/Models/TodoItem.cs

```diff
@@ -0,0 +1,14 @@
+namespace Todo.WillowBear.Models
+{
+
+    public class TodoItem
+    {
+        public int Id { get; set; }
+        public string Title { get; set; } = string.Empty;
+        public int IsDone { get; set; }
+
```

Comment:
It's ok to separate properties in groups with empty lines, but what's the criteria here?

Reply:
Un-intended! Have fixed.